holidays/countries/andorra.py

```diff
@@ -19,10 +21,60 @@ class Andorra(HolidayBase, ChristianHolidays, InternationalHolidays):
 
     References:
         * <https://en.wikipedia.org/wiki/Public_holidays_in_Andorra>
-        * <https://www.holsdb.com/public-holidays/ad>
+        * [Andorra's Constitution](https://web.archive.org/web/20250506145838/https://www.bopa.ad/Legislacio/Detall?doc=7586)
```

Comment:
I don't think any of the wayback machine archives of bopa.ad link works (likely from JavaScript issues), might wanna switch to archive.today for these ones instead i.e.

http://archive.today/2025.05.06-163056/https://www.bopa.ad/Legislacio/Detall?doc=7586

Reply:
Yes, these snapshots don't work. Well, at least I checked archive-links script in real situation :)